lightning/src/ln/channelmanager.rs

```diff
@@ -924,6 +933,19 @@ pub(crate) const MPP_TIMEOUT_TICKS: u8 = 3;
 /// [`OutboundPayments::remove_stale_resolved_payments`].
 pub(crate) const IDEMPOTENCY_TIMEOUT_TICKS: u8 = 7;
 
+/// The maximum number of unfunded channels we can have per-peer before we start rejecting new
+/// (inbound) ones. The number of peers with unfunded channels is limited separately in
+/// [`MAX_UNFUNDED_CHANNEL_PEERS`].
+const MAX_UNFUNDED_CHANS_PER_PEER: usize = 4;
```

Comment:
From what I understand, this check applies to both inbound and outbound peers. Comment could precise it. Same with `MAX_UNFUNDED_CHANNEL_PEERS` I think.

Reply:
Yeah I mean could be nice to add mentions for inbound/outbound connections. Not big thing though.